flow/scene_update_context.cc

```diff
@@ -356,7 +375,7 @@ SceneUpdateContext::Frame::Frame(SceneUpdateContext& context,
       paint_bounds_(SkRect::MakeEmpty()),
       layer_(layer) {
   if (elevation != 0.0)
-    entity_node().SetTranslationRH(0.f, 0.f, -elevation);
+    entity_node().SetTranslationRH(0.f, 0.f, elevation);
```

Comment:
Should probably be -elevation (why the change?)

Note: this would have been "elevation" when Scenic was left-handed, but needs to be "-elevation" now that it's right-handed.

Reply:
I think a bad merge might have caused this.  I started this change when scenic was left-handed, then jiri updated at one point and scenic became right-handed.

This ended up being the source of the disappearing views bug.